keyboards/electronlab/klor/keymaps/electronlab/keymap.c

```diff
@@ -0,0 +1,779 @@
+    _______,  _______,  _______,  _______,  _______,  _______,   _______,  _______,  _______,  _______,  _______,  _______,
+                        _______,  _______,  _______,  _______,   _______,  _______,  _______,  _______
+ )
+
+
+   ╺━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸
+   
+   ┌─────────────────────────────────────────────────┐
+   │ t e m p l a t e   s a e g e w e r k             │
+   └─────────────────────────────────────────────────┘
+   ┌─────────┬─────────┬─────────┬─────────┬─────────┐                    ┌─────────┬─────────┬─────────┬─────────┬─────────┐
+   │         │         │         │         │         │ ╭╮╭╮╭╮╭╮╭╮╭╮╭╮╭╮╭╮ │         │         │         │         │         │   
+   ├─────────┼─────────┼─────────┼─────────┼─────────┤ │╰╯╰╯╰╯╰╯╰╯╰╯╰╯╰╯│ ├─────────┼─────────┼─────────┼─────────┼─────────┤
+   │         │         │         │         │         ├─╯                ╰─┤         │         │         │         │         │
+   ├─────────┼─────────┼─────────┼─────────┼─────────┤╭────────╮╭────────╮├─────────┼─────────┼─────────┼─────────┼─────────┤
+   │         │         │         │         │         ││        ││        ││         │         │         │         │         │
+   └─────────┴─────────┼─────────┼─────────┼─────────┤╰────────╯╰────────╯├─────────┼─────────┼─────────┼─────────┴─────────┘
+                       │         │         │         │                    │         │         │         │  
+                       └─────────┴─────────┴─────────┘                    └─────────┴─────────┴─────────┘ 
+
+   [_TEMPLATE] = LAYOUT_saegewerk(
+ //╷         ╷         ╷         ╷         ╷         ╷         ╷╷         ╷         ╷         ╷         ╷         ╷         ╷
+    _______,  _______,  _______,  _______,  _______,                       _______,  _______,  _______,  _______,  _______,
+    _______,  _______,  _______,  _______,  _______,                       _______,  _______,  _______,  _______,  _______,
+    _______,  _______,  _______,  _______,  _______,  _______,   _______,  _______,  _______,  _______,  _______,  _______,
+                        _______,  _______,  _______,                       _______,  _______,  _______
+ )
+
+ */
+};
+
+// ┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+// │ O L E D                                                                                                                                    │
+// └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+// ▝▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▘
+
+#ifdef OLED_ENABLE
```

Comment:
As above, this should probably be moved to keyboard level.

Reply:
I've put back the enable test for hardware features in config.h at keyboard level